.github/workflows/test.yml

```diff
@@ -25,19 +26,19 @@ jobs:
         with:
           node-version: "${{ matrix.node-version }}"
           cache: npm
-      - run: npm ci
-      - run: "npm run test:ci"
+      - run: npm clean-install
+      - run: npm test
   test:
     runs-on: ubuntu-latest
     needs: test_matrix
     steps:
       - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3
       - uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3
         with:
-          node-version: 16
+          node-version: "lts/*"
           cache: npm
-      - run: npm ci
+      - run: npm clean-install
+      - run: npm audit signatures
       - name: Ensure dependencies are compatible with the version of node
         run: npx ls-engines
       - run: npm run lint
-      - run: npx lockfile-lint --path package-lock.json
```

Comment:
I compared to https://github.com/semantic-release/npm/blob/master/.github/workflows/test.yml which doesn't have it either. Happy to leave it in if you like

Reply:
looks like we [haven't added it consistently](https://github.com/search?q=org%3Asemantic-release+lockfile-lint&type=code), so looks like i dropped the ball on that. i'll add that to my list if you're good with it being in place. 

i see that check as high value since it protects us against PRs where the `package.json` changes for a dependency look valid, but the lockfile (which we're unlikely to review closely) points to an alternative registry, for example.

Reply:
totally agree, I'll add it back 👍🏼 Thank you!